packages/neon/neon_talk/lib/src/widgets/rich_object/mention.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:neon_framework/theme.dart';
+import 'package:neon_framework/widgets.dart';
+import 'package:nextcloud/spreed.dart' as spreed;
+
+/// Displays a mention chip from a rich object.
+class TalkRichObjectMention extends StatelessWidget {
+  /// Create a new Talk rich object mention.
+  const TalkRichObjectMention({
+    required this.parameter,
+    super.key,
+  });
+
+  /// The parameter to display.
+  final spreed.RichObjectParameter parameter;
+
+  @override
+  Widget build(BuildContext context) {
+    final child = switch (parameter.type) {
+      'user' => NeonUserAvatar(
+          username: parameter.id,
+          showStatus: false,
+        ),
+      'call' => CircleAvatar(
+          child: ClipOval(
+            child: NeonUriImage(
+              uri: Uri.parse(parameter.iconUrl!),
+            ),
+          ),
+        ),
+      'guest' => CircleAvatar(
+          child: Icon(AdaptiveIcons.person),
+        ),
+      'user-group' || 'group' => CircleAvatar(
+          child: Icon(AdaptiveIcons.group),
+        ),
+      _ => throw UnimplementedError('Unknown mention type: $parameter'), // coverage:ignore-line
+    };
+
+    return Chip(
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(50)),
+      padding: EdgeInsets.zero,
+      labelPadding: const EdgeInsetsDirectional.only(end: 8),
+      avatar: Padding(
+        padding: const EdgeInsets.all(4),
+        child: child,
+      ),
+      label: Text(parameter.name),
+    );
```

Comment:
The padding is the same as used in the `TalkRichObjectFallback` widget.
Please add them to a `TalkTheme` ThemeExtension that can be registered in the AppImplementation:
https://github.com/nextcloud/neon/blob/70eb554f1e9a9fa079e5365a2f0eb871296e9700/packages/neon_framework/lib/src/models/app_implementation.dart#L192

Reply:
I'll wait for https://github.com/nextcloud/neon/pull/1849

Reply:
The chips have diverged now to make the mentions in line with the text it would be embedded in and they don't share any common values anymore.